cassandra/deserializers.pyx

```diff
@@ -127,8 +127,8 @@ cdef class DesCounterColumnType(DesLongType):
 
 cdef class DesDateType(Deserializer):
     cdef deserialize(self, Buffer *buf, int protocol_version):
-        cdef double timestamp = unpack_num[int64_t](buf) / 1000.0
```

Comment:
Nice analysis. Thanks for enlightening us.
Do you think we should try to align the cython version with CPython at this point, or just fix the overflow issue for this ticket, and have a follow-on issue to align (possibly by going to integer math on the Python side)?